var/spack/repos/builtin/packages/opencarp/package.py

```diff
@@ -66,7 +67,7 @@ class Opencarp(CMakePackage):
     depends_on("py-carputils", when="+carputils", type=("build", "run"))
```

Comment:
The above dep types seem odd. We're linking to the pkgconfig/python/perl libs? Should they be type=build? Or run?